code/game/gamemodes/clock_cult/clock_scriptures/scripture_drivers.dm

```diff
@@ -170,7 +170,10 @@
 	return target && binding && target.buckled == binding && !is_servant_of_ratvar(target) && target.stat != DEAD
 
 /datum/clockwork_scripture/geis/scripture_effects()
-	return add_servant_of_ratvar(target)
+	. = add_servant_of_ratvar(target)
+	if(.)
+		target.log_message("<font color=#BE8700>Conversion was done with Geis, by [invoker.name][invoker.ckey ? "([invoker.ckey])":""].</font>", INDIVIDUAL_ATTACK_LOG)
+		invoker.log_message("<font color=#BE8700>Converted [target.name][target.ckey ? "([target.ckey])":""] with Geis.</font>", INDIVIDUAL_ATTACK_LOG)
```

Comment:
Converting someone leaving the same color as being converted is bad. Make the logging use the standard yellow/red for attacks or just remove colors completly.